### PR TITLE
Create gradle.yml

### PR DESCRIPTION
**Pull request by tadhgryan2012**
Adding an action to the Pipeline to test the gradlew build command

### CODE REVIEW

File: .github/workflows/gradle.yml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+    # Configure Gradle for optimal use in GitHub Actions, including caching of downloaded dependencies.
+    # See: https://github.com/gradle/actions/blob/main/setup-gradle/README.md
+    - name: Setup Gradle
+      uses: gradle/actions/setup-gradle@af1da67850ed9a4cedd57bfd976089dd991e2582 # v4.0.0
+      with:
+        gradle-version: '8.10.2'
+ 
+    - name: Build with Gradle Wrapper
+      run: ls
```
Comment (oOoOoOo0o) [inline .github/workflows/gradle.yml:39]: What purpose does running ls serve? Also if we are just running ls the name of the task should reflect that.

Comment (tadhgryan2012) [reply .github/workflows/gradle.yml:39]: this is a wip

**Review by s1m0nWasTaken — DISMISSED** [review]
Seems to be passing 👍

File: .github/workflows/gradle.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+jobs:
+  gradle:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - uses: actions/setup-java@v4
+      with:
+        distribution: temurin
+        java-version: 17
```
Comment (oOoOoOo0o) [inline .github/workflows/gradle.yml:14]: In other areas of our project we have specified the Java version to be 21, is it supposed to be 17 here?

Comment (tadhgryan2012) [reply .github/workflows/gradle.yml]: My bad, missed that

**Review by s1m0nWasTaken — APPROVED** [review]

**Review by oOoOoOo0o — APPROVED** [review]
Looking good

**Pull request merged**